ticket_group_details.view.lkml

```diff
@@ -0,0 +1,60 @@
+view: ticket_group_details {
+  derived_table: {
+    sql: SELECT
+        DISTINCT audits.ticket_id AS ticket_id,
+        groups.name AS first_group,
+        events.value AS first_group_id
+      FROM
+        zendesk_stitch.ticket_audits__events AS events
+      LEFT JOIN
+        zendesk_stitch.ticket_audits AS audits
+      ON
+        audits.id = events._sdc_source_key_id
+      LEFT JOIN
+        zendesk_stitch.groups AS groups
+      ON
+        groups.id::text = events.value
+      WHERE
+        events.field_name = 'group_id'
+        AND events.type = 'Create'
+       ;;
+    indexes: ["ticket_id"]
+    sql_trigger_value: SELECT COUNT(*) FROM zendesk_stitch.ticket_audits__events ;;
+  }
+
+  dimension: ticket_id {
+    primary_key: yes
+    type: number
+    hidden: yes
+    sql: ${TABLE}.ticket_id ;;
+  }
+
+  dimension: first_group {
+    description: "The name of the first group the ticket was assigned to"
+    group_label: "First Touch"
+    type: string
+    sql: ${TABLE}.first_group ;;
+  }
+
+  dimension: last_group {
+    description: "The name of the last group the ticket was assigned to"
+    group_label: "Last Touch"
+    type: string
+    sql: ${tickets.group_name} ;;
+  }
+
+  dimension: first_group_id {
```

Comment:
No `last_group_id`?

Reply:
I decided not to include as it would just be a dimension that we would be hidden.  The last_group_id would always be the same as the current group_id